### PR TITLE
Blog app - controllers specs

### PR DESCRIPTION
**Pull request by mohametalmeari**
- [ ] Add RSpec as a gem to your project
- [ ] Create a Request spec file for all controllers
- [ ] Make sure that each action checks:
  - If response status was correct
  - If a correct template was rendered
  - If the response body includes correct placeholder text

### CODE REVIEW

File: spec/requests/posts_spec.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  end
+
+  describe 'GET /show' do
+    it 'returns http success' do
+      get '/users/13/posts/7'
+      expect(response.body).to include('<h2>Post</h2>')
+      expect(response.status).to eq(200)
+    end
+  end
+end
```
Comment (mckent05) [inline spec/requests/posts_spec.rb:18]: - Good job done so far, but according to the project requirements, please write tests that ensures correct view template is rendered. Please fix this for both `index and show` views 🙏

File: spec/requests/users_spec.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  end
+
+  describe 'GET /show' do
+    it 'returns http success' do
+      get '/users/13'
+      expect(response.body).to include('<h2>User\'s Page</h2>')
+      expect(response.status).to eq(200)
+    end
+  end
+end
```
Comment (mckent05) [inline spec/requests/users_spec.rb:18]: - - Good job done so far, but according to the project requirements, please write tests that ensure the correct view template is rendered. Please fix this for both `index and show` views 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @mohametalmeari, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- Good job adding test for your controllers 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/users_spec.rb (hunk 2 above)
Comment (mohametalmeari) [reply spec/requests/users_spec.rb]: Hi @mckent05, thanks for the review, I added the required test

**Review by mckent05 — APPROVED** [review]
Hi @mohametalmeari, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
- Good job adding tests 👏 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**